### ISSUE TITLE
Feature request: copy comments

### ISSUE THREAD

**Issue by tkluysk** [ericyd/gdrive-copy#40]
GDrive now allows for copying with comments, and even suggestions:

![api_test_-_google_docs](https://user-images.githubusercontent.com/555583/37439067-fa14cab4-285a-11e8-89ad-46f6dd283dc7.png)

I had a quick look at the API, and unfortunately that feature is not directly exposed:
https://developers.google.com/drive/v3/reference/files/copy

It can be done though.  Other Chrome extensions do exactly that (but don't deep copy like this script).

https://webapps.stackexchange.com/questions/47732/google-apps-script-copy-document-comments

**Comment by ericyd** [comment]
Yeah looks like I would need to use the [Comments API](https://developers.google.com/drive/v2/reference/comments) to get current comments and copy to new file.

I would put this in the "not impossible but not trivial" category.  I think the major downside would be increased API usage - one doc could potentially have hundreds of comments.

**Comment by tkluysk** [comment]
The developer of CopyComments has been so kind to put up his source code on my request:
https://github.com/jsonchin/Copy-Comments

He seems to have some throttling/retry code in there.
